orderfilter/filter_js.go

```diff
@@ -24,13 +24,13 @@ func New(chainID int, customOrderSchema string, contractAddresses ethereum.Contr
 	chainIDSchema := fmt.Sprintf(`{"$id": "/chainId", "const":%d}`, chainID)
 	exchangeAddressSchema := fmt.Sprintf(`{"$id": "/exchangeAddress", "enum":[%q,%q]}`, contractAddresses.Exchange.Hex(), strings.ToLower(contractAddresses.Exchange.Hex()))
 
-	if jsutil.IsNullOrUndefined(js.Global().Get("createSchemaValidator")) {
-		return nil, errors.New(`"createSchemaValidator" has not been set on the Javascript "global" object`)
+	if jsutil.IsNullOrUndefined(js.Global().Get("__mesh_createSchemaValidator__")) {
```

Comment:
Are you talking about making `js.Global()` a constant? I don't really think that saves us anything.

Reply:
No, I’m suggesting that we make the string “__mesh_createSchemaValidator__” a string. Not a big deal, but it would be nice IMO

Reply:
I went with a different approach here. Instead of defining constants, I just made a `_setGlobals` function that we call. This ensures the process and global variable names are exactly the same in `@0x/mesh-browser`, `@0x/mesh-browser-lite`, and __./packages/wasm-test__.